tests/commands/ostreesetup.py

```diff
@@ -70,5 +70,32 @@ class RHEL7_TestCase(F21_TestCase):
     def runTest(self):
         F21_TestCase.runTest(self)
 
+class F38_TestCase(F21_TestCase):
+    def runTest(self):
+        # Check default arguments settings
+        cmd = F38_OSTreeSetup()
+        self.assertFalse(cmd.nogpg)
+        self.assertFalse(cmd.bootc)
+
+        # no remote, should equal osname (check for compatibility)
+        cmd.parse('--osname=fedora-atomic --url=http://example.com/repo --ref=fedora-atomic/sometest/base/core'.split(' '))
+        self.assertEqual(cmd.remote, cmd.osname)
+
+        # check backward compatibility
+        F21_TestCase.runTest(self)
+
+        # pass
+        self.assert_parse("ostreesetup --bootc --url=quay.io/fedora/coreos:stable")
```

Comment:
This doesn't check the generated string since you didn't specified the expected value for comparison.